collection_requirements.rst

```diff
@@ -115,16 +115,7 @@ galaxy.yml
 ----------
 
 * The ``tags`` field MUST be set.
-* Collection dependencies must have a lower bound on the version which is at least 1.0.0.
-
-  * This means that all collection dependencies have to specify lower bounds on the versions, and these lower bounds should be stable releases, and not versions of the form 0.x.y.
-  * When creating new collections where collection dependencies are also under development, you need to watch out since Galaxy checks whether dependencies exist in the required versions:
-
-    #. Assume that ``foo.bar`` depends on ``foo.baz``.
-    #. First release ``foo.baz`` as 1.0.0.
-    #. Then modify ``foo.bar``'s ``galaxy.yml`` to specify ``'>=1.0.0'`` for ``foo.baz``.
-    #. Finally release ``foo.bar`` as 1.0.0.
-
+* Collection dependencies must satisfy a set of rules. See the section on `Collection Dependencies <collection_dependencies_>`_ for this.
```

Comment:
Collection dependencies must meet a set of rules. See the section on Collection Dependencies <collection_dependencies_>_ for this.